doc/lua_api.txt

```diff
@@ -2283,6 +2283,9 @@ Elements
   The axes are euler angles in degrees.
 * `continuous` (Optional): Whether the rotation is continuous. Default `false`.
 * `mouse control` (Optional): Whether the model can be controlled with the mouse. Default `true`.
+* `frame loop range` (Optional): Range of the animation frames.
+  * Defaults to the full range of all available frames.
+  * Syntax: `<begin>,<end>`; example: `model[0,0;4,6;player_preview;character.b3d;character.png;0,180;false;true;0,79]`
```

Comment:
Probably shouldn't have an example; no other property has an example, and I think the syntax is clear enough to not need one.  The example is just making things longer.